awx/main/middleware.py

```diff
@@ -192,6 +212,10 @@ def _convert_named_url(self, url_path):
                                                  url_units[4])
         return '/'.join(url_units)
 
+    def __call__(self, request):
+        response = self.process_request(request)
+        return response
```

Comment:
I'm confused here. I expected to see this in `MigrationRanCheckMiddleware` because it short-circuits in many cases.

But this class should never short-circuit, should it? It only modifies the request. Because of that, I would expect to see the standard pattern...


```python
    def __call__(self, request):
        # Code to be executed for each request before
        # the view (and later middleware) are called.

        response = self.get_response(request)

        # Code to be executed for each request/response after
        # the view is called.

        return response
```

It's weird that `process_request` is where `get_response` should be, even if it ultimately calls that. It doesn't make it clear, at a glance, that `URLModificationMiddleware` modifies the request, but never hijacks the rest of the stack.

Reply:
Fixed it!